.github/workflows/deploy.yml

```diff
@@ -8,24 +8,40 @@ on:
   push:
     branches:
       - main
+  schedule:
+    # Run weekly at 00:00 on Sunday
+    - cron: '0 0 * * 0'
 
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
     steps:
       - name: Cancel Previous Runs
         uses: styfle/cancel-workflow-action@0.9.1
 
       - name: Checkout
         uses: actions/checkout@v2.3.5
 
-      - name: Set up Python
-        uses: actions/setup-python@v2
+      - name: Setup Mambaforge
+        uses: conda-incubator/setup-miniconda@v2.1.1
+        with:
+          activate-environment: seismo-learn
+          environment-file: environment.yml
+          miniforge-variant: Mambaforge
+          use-mamba: true
+
+      - uses: actions/cache@v2
+        id: cache
+        with:
+          path: build/.jupyter_cache
+          key: notebooks
 
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install -r requirements.txt
+      - name: Disable notebook cache for scheduled jobs
+        if: github.event_name == 'schedule'
+        run: sed -i"" 's/jupyter_execute_notebooks = "cache"/jupyter_execute_notebooks = "force"/' source/conf.py
```

Comment:
`-i""` or `-i`?

Reply:
`-i""` works for both GNU sed and BSD sed, while `-i` only works for GNU seed.